pkg/ddevapp/compose_yaml.go

```diff
@@ -102,6 +102,11 @@ func fixupComposeYaml(yamlStr string, app *DdevApp) (map[string]interface{}, err
 		return nil, err
 	}
 
+	dockerIP, _ := dockerutil.GetDockerIP()
```

Comment:
nit here: IMO it would be better to use a variable name like `bindIP` here, to describe what it's actual purpose is.